kim/field.py

```diff
@@ -353,12 +357,14 @@ def __init__(self, **kwargs):
 
         :param max: Specify the maximum permitted length
         :param min: Specify the minimum permitted length
+        :param blank: If False, raise error if empty string passed. Default True
```

Comment:
It is limited to strings. I decided this after checking what it's like in Django. I think it would be a bad idea to cause confusion with a library most people are going to be familiar with.

I'm not sure that preventing falsy values other than the empty string really has any value, I couldn't think of any genuine use cases that are not otherwise covered by eg min/max params.